load_tests/bar.py

```diff
@@ -0,0 +1,25 @@
+
```

Comment:
I would prefer to call this file as `benchmark_bar.py` or `validation_bar.py`

Reply:
changed.